src/main/java/edu/ucsb/cs56/ucsb_courses_search/formbeans/ScheduleSearch.java

```diff
@@ -0,0 +1,25 @@
+package edu.ucsb.cs56.ucsb_courses_search.formbeans;
+
+public class ScheduleSearch {
+    private int scheduleid;
```

Comment:
`scheduleid` doesn't have type `int`, it has type `long`. If we exceed the maximum integer value, this will fail.

Reply:
We ended up never using formbeans at all so we planned to delete this in the last PR but will still need to